include/wsrep/provider.hpp

```diff
@@ -519,10 +521,9 @@ namespace wsrep
         static std::unique_ptr<provider> make_provider(
             wsrep::server_state&,
             const std::string& provider_spec,
-            const std::string& provider_options,
+            const std::function<std::string(provider_options&)>& provider_options_cb,
```

Comment:
Could `provider_options` be const reference?

Fix documentation to reflect the signature change.

Reply:
Fixed